astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -88,76 +98,95 @@ class galactocentric_frame_defaults(ScienceState):
     _references = None
     _value = None
 
+    _registry = {
+        "v4.0": {
+            "value": MappingProxyType(
+                {
+                    "galcen_coord": ICRS(
+                        ra=266.4051 * u.degree, dec=-28.936175 * u.degree
+                    ),
+                    "galcen_distance": 8.122 * u.kpc,
+                    "galcen_v_sun": r.CartesianDifferential(
+                        [12.9, 245.6, 7.78] * u.km / u.s
+                    ),
+                    "z_sun": 20.8 * u.pc,
+                    "roll": 0 * u.deg,
+                }
+            ),
+            "references": MappingProxyType(
+                {
+                    "galcen_coord": "http://adsabs.harvard.edu/abs/2004ApJ...616..872R",
+                    "galcen_distance": "https://ui.adsabs.harvard.edu/abs/2018A%26A...615L..15G",
+                    "galcen_v_sun": [
+                        "https://ui.adsabs.harvard.edu/abs/2018RNAAS...2..210D",
+                        "https://ui.adsabs.harvard.edu/abs/2018A%26A...615L..15G",
+                        "https://ui.adsabs.harvard.edu/abs/2004ApJ...616..872R",
+                    ],
+                    "z_sun": "https://ui.adsabs.harvard.edu/abs/2019MNRAS.482.1417B",
+                    "roll": None,
+                }
+            ),
+        },
+        "pre-v4.0": {
```

Comment:
Might it make sense for the outer one to be a `MappingProxyType` too?

Reply:
I refrained from doing that so that the pre-registered options are editable by enterprising users who want to maintain consistency between their added metadata and the pre-registered options. For instance if a new option "test" with extra field "stdv" is added, something can be added to "v4.0" or "pre-v4.0" for consistency.

> If metadata is added to one parameter set, for consistency it should be added to all. Though the built-in options are read-only for "value" and "references", they permit additional fields, such as the standard deviation.

> ```python
> galactocentric_frame_defaults._registry["v4.0"]["stdv"] = None
> ```

Reply:
OK, makes sense.